actionpack/test/dispatch/routing_test.rb

```diff
@@ -3303,7 +3260,7 @@ def test_explicitly_avoiding_the_named_route
       scope as: "routes" do
         get "/c/:id", as: :collision, to: "collision#show"
         get "/collision", to: "collision#show"
-        get "/no_collision", to: "collision#show", as: nil
+        get "/no_collision", to: "collision#show", as: false
```

Comment:
This might be a regression. The previous behaviour is `get "/no_collision", to: "collision#show", as: nil` wouldn't draw a route and now it will because keyword defaults are nil.

Reply:
Fixed in the latest commit.